Description: Fix FTBFS when GCC configured with --enable-default-pie
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1579023
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2016-05-20
--- a/gpxe/src/Makefile
+++ b/gpxe/src/Makefile
@@ -4,7 +4,7 @@
 #
 
 CLEANUP		:=
-CFLAGS		:=
+CFLAGS		:= -fno-pie
 ASFLAGS		:=
 LDFLAGS		:=
 MAKEDEPS	:= Makefile
